cypress/integration/journeys/add-euss.spec.js

```diff
@@ -1,55 +1,63 @@
 
-            cy.get('[data-testid=callback-form-validation-error]').should('not.exist');
-        });
+        cy.get('[data-testid=callback-form-validation-error]').should('not.exist');
     });
+});
 
-    context('When required fields are not filled in', () => {
-        it('displays validation error if form is submitted with no inputs', () => {
-            cy.get('[data-testid=callback-form-update_button]').click({ force: true });
-            cy.get('[data-testid=callback-form-validation-error]').should('be.visible');
-            cy.url().should('match', /\/add-support/);
-        });
+context('When required fields are not filled in', () => {
+    it('displays validation error if form is submitted with no inputs', () => {
+        cy.get('[data-testid=callback-form-update_button]').click({ force: true });
+        cy.get('[data-testid=callback-form-validation-error]').should('be.visible');
+        cy.url().should('match', /\/add-support/);
+    });
 
-        it('displays validation error when call completed but no follow up selected', () => {
-            cy.get('[data-testid=call-type-radio-button]').eq(5).click({ force: true });
-            cy.get('[data-testid=call-type-yes-radio-button]').click({ force: true });
-            cy.get('[data-testid=yes-spoke-to-resident]').click({ force: true });
-            cy.get('[data-testid=callback_complete-checkbox]').click({ force: true });
-            cy.get('[data-testid=call-direction-radio-button]').first().click({ force: true });
-            cy.get('[data-testid=callback-form-update_button]').click({ force: true });
-            cy.get('[data-testid=callback-form-validation-error]').should('be.visible');
-        });
+    it('displays validation error when call completed but no follow up selected', () => {
+        cy.server();
+        cy.route('GET', '/api/proxy/help-types').as('getHelpTypes');
+        cy.wait('@getHelpTypes', { timeout: 10000});
+        cy.get('[data-testid=call-type-radio-button]').eq(5).click({ force: true });
+        cy.get('[data-testid=call-type-yes-radio-button]').click({ force: true });
+        cy.get('[data-testid=yes-spoke-to-resident]').click({ force: true });
+        cy.get('[data-testid=callback_complete-checkbox]').click({ force: true });
+        cy.get('[data-testid=call-direction-radio-button]').first().click({ force: true });
+        cy.get('[data-testid=callback-form-update_button]').click({ force: true });
+        cy.get('[data-testid=callback-form-validation-error]').should('be.visible');
     });
-}
-if (!IS_EUSS_ENABLED) {
-    context('When EUSS is not enabled', () => {
-        it('does not display EUSS call type option', () => {
-            cy.get('[data-testid=call-type-radio-button]').last().should('not.have.value', EUSS);
-            cy.get('[data-testid=call-type-radio-button]').should('not.contain.value', EUSS);
-        });
+});
+
+context('When EUSS is not enabled', () => {
```

Comment:
I realised that this context should probably be 'When a non-EUSS user is logged in' or something similar. This is just a suggestion though and won't affect the functionality